test/integration/node-specific/operation_examples.test.ts

```diff
@@ -1655,7 +1669,7 @@ describe('Operations', function () {
 
       /* eslint-disable */
 
-      return client.connect().then(function (client) {
+      return client.connect().then(function(client) {
```

Comment:
I feel that while we are reformatting here, we should probably:

1. Make this entire sequence use async/await
2. Fix the calls to `rename()` - it's an async function and its second argument is an options object, and does not take a callback so the signature is wrong in these tests.